### DISCUSSION TITLE
老师是否有使用lazy.nvim的计划

### DISCUSSION THREAD

**Discussion by zhuwj03** [nshen/learn-neovim-lua#170] [Q&A]
老师是否有使用lazy.nvim的计划

**Comment by nshen** [comment]
暂时不太想换，我用 packer.nvim 没有遇到什么问题，感觉懒加载之类没什么用。加上insisVim里对packer.nvim 做了包装，不太想改了。

**Comment by jonahfang** [comment]
lazy.nvim现在大火，不使用的话可能会影响本项目的形象!

**Comment by nshen** [comment]
我把 [InsisVim](https://github.com/nshen/InsisVim) 改成 lazy.nvim 了

https://github.com/nshen/learn-neovim-lua/assets/181506/714336d5-d963-49df-9745-35b84b9212f9
